server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Attachment.java

```diff
@@ -135,7 +135,7 @@ public long getSize() {
         return cid;
     }
 
-    public boolean isInline() {
+    public boolean isIsInline() {
```

Comment:
Yes, explaination:
Jackson removes the verb from the accessors (get, is ...) in order to producer the JSON.
The JMAP spécification is waiting for "isInline": "..."
So we have to double the 'is' verb.
You can found similar method in the Message class.